### PR TITLE
Feat 3858 - add loader before displaying data and code cleaning

### PR DESCRIPTION
**Pull request by phpgeekfr**
factorized wishlist and home component + added loader

### CODE REVIEW

File: apps/festival/festival/src/app/dashboard/title/list/list.component.scss [hunk 1]
```diff
@@ -30,11 +30,6 @@ mat-card {
   }
 }
 
-p {
-  text-align: center;
-  margin-bottom: 32px;
-}
-
 h1 {
```
Comment (fritzschoff) [inline apps/festival/festival/src/app/dashboard/title/list/list.component.scss:33]: I guess you can remove that as well

File: apps/financiers/financiers/src/app/dashboard/title/list/list.component.scss [hunk 2]
```diff
@@ -30,11 +30,6 @@ mat-card {
   }
 }
 
-p {
-  text-align: center;
-  margin-bottom: 32px;
-}
-
 h1 {
```
Comment (fritzschoff) [inline apps/financiers/financiers/src/app/dashboard/title/list/list.component.scss:33]: can't find a h1 in the template

File: libs/ui/src/lib/dashboard/pages/home/home.component.html [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+  <ng-template #noAcceptedMovies>
+    <section fxLayout="column" fxLayoutAlign="center center">
+      <ng-container *ngIf="(hasMovies$ | async) else noMovies">
+        <h1>You have no title published yet.</h1>
+        <p class="mat-body-1">
+          As long as you have no title to display, your organization will not appear on the marketplace.
+          <br>
+          You can either add several titles at once through bulk import, or just one at a time.
+          <br>
```
Comment (fritzschoff) [inline libs/ui/src/lib/dashboard/pages/home/home.component.html:29]: one br is enough?

File: libs/ui/src/lib/marketplace/pages/wishlist/wishlist.component.html [hunk 4]
```diff
@@ -0,0 +1,73 @@
+<section *ngIf="isDataLoaded; else loading">
+  <ng-container *ngIf="hasWishlist; else noWishlist">
+<article fxLayout="row" fxLayoutAlign="start center">
```
Comment (fritzschoff) [inline libs/ui/src/lib/marketplace/pages/wishlist/wishlist.component.html:3]: isn't h1 a block element and if you need to center it just put text-align: center?

File: libs/ui/src/lib/marketplace/pages/wishlist/wishlist.component.html [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+      {{ element.title.international }}
+    </td>
+  </ng-container>
+
+  <!-- Director -->
+  <ng-container matColumnDef="director">
+    <th mat-header-cell *matHeaderCellDef>Director</th>
+    <td mat-cell *matCellDef="let element">
+      <span *ngFor="let director of element.directors | slice: 0:2; let isLast = last">
+        {{ director.firstName | titlecase }} {{ director.lastName | titlecase }}{{ isLast ? '' : ', ' }}
```
Comment (fritzschoff) [inline libs/ui/src/lib/marketplace/pages/wishlist/wishlist.component.html:20]: use the displayName pipe and remove isLast. Francois did a comment about that in dev

**Review by fritzschoff — APPROVED** [review]

File: apps/festival/festival/src/app/dashboard/title/list/list.component.scss (hunk 1 above)
Comment (phpgeekfr) [reply apps/festival/festival/src/app/dashboard/title/list/list.component.scss:33]: Have a second look ;)

File: apps/financiers/financiers/src/app/dashboard/title/list/list.component.scss (hunk 2 above)
Comment (phpgeekfr) [reply apps/financiers/financiers/src/app/dashboard/title/list/list.component.scss:33]: Have a second look ;)

**Pull request merged**